react/src/machine-coding/progressbar/ProgressBar.module.css

```diff
@@ -38,6 +38,13 @@
   border: none;
 }
 
+@media (max-width: 768px) {
+  .controls {
+    width: 100%;
+    gap: 1rem;
+  }
+}
+
```

Comment:
Use a mobile-first approach. Write the styles for mobile and then overwrite the styles for bigger screens in media queries. You would want to use `min-width` instead of `max-width` for this.